logging/src/test/groovy/life/qbic/logging/impl/publisher/SimplePublisherSpec.groovy

```diff
@@ -0,0 +1,56 @@
+package life.qbic.logging.impl.publisher
+
+import life.qbic.logging.api.LogLevel
+import life.qbic.logging.api.LogMessage
+import life.qbic.logging.api.Subscriber
+import spock.lang.Specification
+
+import java.nio.charset.Charset
+
+/**
+ * <b><class short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
+class SimplePublisherSpec extends Specification {
```

Comment:
There is no test for the other log levels and a throwable payload. Is this by design?